### DISCUSSION TITLE
TVP VOD plugin

### DISCUSSION THREAD

**Discussion by Irek7** [streamlink/streamlink#4907] [Q&A] [answered]
Is it possible to add support for VOD of tvp.pl?
They provide two types of materials:
1. https://vod.tvp.pl/informacje-i-publicystyka,205/woronicza-17-odcinki,273788
2. https://www.tvp.info/40374360/program-minela-9

**Comment by bastimeyer** [comment, marked as answer]
Since #4905 hasn't been merged yet, I decided to have a quick look. Implementation was fairly simple, so I updated the PR.

**Reply by bastimeyer** [reply]
Btw, please don't use the discussion forum for plugin (improvement) requests
